packages/devtools_app/lib/src/common_widgets.dart

```diff
@@ -492,6 +492,23 @@ class ToolbarAction extends StatelessWidget {
   }
 }
 
+/// A [ValueListenable] that ignores all added listeners and returns a fixed
+/// value.
+class FixedValueListenable<T> extends ValueListenable<T> {
```

Comment:
I like it! 
Move it to a separate file for value listenable helpers as this isn't a Widget.
Maybe:
`listenable.dart`
I expect at some point we will add a helper for a List that may have its elements change.